.github/workflows/python-api-test-workflows.yml

```diff
@@ -0,0 +1,17 @@
+name: Python api-test workflows
+
+on:
+  pull_request:
+    types:
+      - opened
+      - closed
+      - synchronize
+
+jobs:
```

Comment:
This has plenty of python code but does not include the Python lint and testing Github Workflows

Reply:
https://github.com/ai-cfia/github-workflows/blob/main/.github/workflows/workflow-lint-test-python.yml

Reply:
Done